### PR TITLE
Dockerfile fixes and improvements 

### PR DESCRIPTION
**Pull request by sappo**

### CODE REVIEW

File: zproject_known_projects.xml [hunk 1]
```diff
@@ -33,7 +33,7 @@
         </use>
     -->
 
-    <use project = "libzmq" prefix = "zmq" debian_name = "libzmq3-dev" redhat_name = "zeromq-devel"
+    <use project = "libzmq" prefix = "zmq" debian_name = "libzmq5-dev" redhat_name = "zeromq-devel"
```
Comment (bluca) [inline zproject_known_projects.xml]: the old name is the correct one

Comment (bluca) [reply zproject_known_projects.xml]: https://sources.debian.org/src/zeromq3/4.3.2-1/debian/changelog/#L157

Comment (sappo) [reply zproject_known_projects.xml]: I see libzmq5-dev is used in latest Ubuntu though. Maybe we need to differentiate between them.

Comment (bluca) [reply zproject_known_projects.xml]: it's not used, it's just a metapackage - https://tracker.debian.org/media/packages/z/zeromq3/control-4.3.2-1

**Comment by sappo** [comment]
@bluca I removed the commit changing the debian package.

**Pull request merged**